README.md

```diff
@@ -81,3 +81,35 @@ Virtualbox - https://www.virtualbox.org/
 Vagrant - https://www.vagrantup.com/
 
 
+## Docker 
+
+1. To build the image 
+```
+$ docker build --tag todo-app .
```

Comment:
I would always be explicit with the target stage to build, like you are in your step `4.`.
Without `--target`, you are relying on the order of the stages in the Dockerfile, which could cause surprising behaviour.